docs/blog/posts/validation-part1.md

```diff
@@ -298,24 +314,22 @@ If we create a `AIResponse` instance with an answer that does not follow the cha
     [type=value_error, input_value={'chain_of_thought': '1 +... meaning of life is 42'}, input_type=dict]
 ```
 
-Beyond validating multiple attributes of a model, we can also introduce context to our validation functions, in order to give our models more information to work with.
-
 ### Validating Citations From Original Text
 
-Let's see a more concrete example. Let's say we use RAG to answer a question given a text chunk, in earlier systems we'd just simply tell you the chunk id and you'd have to go and find the text chunk yourself. But what if we could validate that the answer is actually supported by the text chunk? We can do this by passing the text chunk as a context to the validator.
-
-Pydantic allows us to do so easily by utilising a context object. This is an arbitrary dictionary which you can access inside the `info` argument in a decorated validator function.
+Let's see a more concrete example. Let's say that we've asked our model a question about some text source and we want to validate that the generated answer is supported by the source. While we could verify this by looking up the original source manually, a more scalable approach is to use a validator to do this automatically.
```

Comment:
```suggestion
Let's see a more concrete example. Let's say that we've asked our model a question about some text source and we want to validate that the generated answer is supported by the source. This would allow us to minimize hallucinations and prevent statements that are not backed by the original text. While we could verify this by looking up the original source manually, a more scalable approach is to use a validator to do this automatically.
```